docs/docs/static-folder.md

```diff
@@ -17,25 +17,14 @@ You can create a folder named `static` at the root of your project. Every file y
 
 ### Referencing your static asset
 
-In order to reference assets from the `static` folder in your code, you'll need to [import the `withPrefix` helper function from `gatsby`](/docs/gatsby-link/#prefixed-paths-helper):
+You can reference assets from the `static` folder in your code without anything special required:
 
 ```js
-import { withPrefix } from 'gatsby'
-
 render() {
   // Note: this is an escape hatch and should be used sparingly!
```

Comment:
Ist this note still valid?

Reply:
Or why we describing an escape hatch instead of an "normal import"

Reply:
The language here is subjective, I chose to keep it in to describe it as something you should try to avoid rather than painting imports as "more normal" (normal usually isn't the correct word, as it means something different to everyone)